pyproject.toml

```diff
@@ -0,0 +1,41 @@
+[tool.poetry]
+name = "MoUSE"
+version = "1.0.0"
+description = "Joseph the MoUSE: Application for rodent ultrasonic identification - backend. Described in: Kania, Adam, et al. \"Joseph the MoUSE—Mouse Ultrasonic Sound Explorer.\" SoftwareX 25 (2024): 101606."
+authors = [ "Leszek Grzanka <https://github.com/grzanka>",
```

Comment:
thats not in the form poetry requires:

![image](https://github.com/user-attachments/assets/be41644d-c2aa-4a58-9f4d-6253aa93672d)